PYME/Analysis/tile_pyramid.py

```diff
@@ -254,7 +254,39 @@ def flush(self):
     def __del__(self):
         self._cur.close()
         self._conn.close()
-    
+
+
+TILEIO_EXT = {
+    '.pzf': PZFTileIO,
+    '.npy': NumpyTileIO,
+    '.db': SqliteTileIO,
+}
+
+def get_tileio_backend(base_directory):
```

Comment:
```suggestion
def infer_tileio_backend(base_directory):
```

Subtle change to make it clear that we are guessing the backend / rather than specifying